src/modules/table.marko

```diff
@@ -0,0 +1,969 @@
+<div id="table">
+    <section-header metadata=input.metadata name=input.name alpha="true"/>
+
+    <p>A tables represents tabular data in rows and columns with incrementally enhanced feature set added on top for sorting, editing, etc.</p>
+
+    <p>Interactive elements should be tabbed into in correct sequence top down row by row and column by column based on the respective LTR or RTL mode.</p>
+
+    <h3>Basic Default Table</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="table">
```

Comment:
Is the `<div>` root tag necessary? Or could the root just be `<table>`?

Reply:
Yes, the wrapping `div` seems necessary. There's a lot more background that I uncovered during the POC I did that I can't recall. All the POC examples included it. One of the bigger reasons it's necessary is for scrolling. Without it, tables don't scroll very well and we lose a lot of flexibility in potentially useful features, such as column freezing.